alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestIndexMonitorAction.kt

```diff
@@ -101,22 +107,24 @@ class RestIndexMonitorAction(
         val xcp = request.contentParser()
         ensureExpectedToken(Token.START_OBJECT, xcp.nextToken(), xcp::getTokenLocation)
         val monitor = Monitor.parse(xcp, id).copy(lastUpdateTime = Instant.now())
-        val monitorVersion = RestActions.parseVersion(request)
+        val seqNo = request.paramAsLong(IF_SEQ_NO, SequenceNumbers.UNASSIGNED_SEQ_NO)
```

Comment:
Has the Kibana plugin been updated to send these parameters instead of `version`?

Reply:
@vinooamzn Yes, it is currently undergoing [PR](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/pull/44/files).